scripts/spack/configs/config.yaml

```diff
@@ -19,7 +19,7 @@ config:
   install_tree:
     root: $spack/..
     projections:
-     all: "${COMPILERNAME}-${COMPILERVER}/${PACKAGE}-${VERSION}"
+      all: "${ARCHITECTURE}/${COMPILERNAME}-${COMPILERVER}/${PACKAGE}-${VERSION}-${HASH}"
```

Comment:
`ARCHITECTURE` is handled by the build scripts and this will duplicate that

Reply:
Removed